packages/web3/src/connect-button/connect-button.tsx

```diff
@@ -84,7 +84,21 @@ export const ConnectButton: React.FC<ConnectButtonProps> = (props) => {
 
   const chainSelect = renderChainSelect();
 
-  const buttonInnerText = (
+  const buttonInnerText = avatar ? (
+    <Space className="flex-row justify-content-center">
+      {restProps?.icon ? (
+        <>
+          <div className={classNames(`${prefixCls}-text`, hashId)}>{buttonText}</div>
+          <Avatar {...avatar} size="small" />
```

Comment:
Try to better match the 'size' attribute of ConnectButton, add a demo to see the effect, and optimize it a bit.

Reply:
Now avatar will display according to size of connect button.